packages/framework-provider-local/src/index.ts

```diff
@@ -20,17 +20,35 @@ import {
 } from './library/read-model-adapter'
 import { searchEntitiesIds, searchEvents } from './library/events-search-adapter'
 import { rawScheduledInputToEnvelope } from './library/scheduled-adapter'
+import {
+  deleteConnectionData,
+  fetchConnectionData,
+  sendMessageToConnection,
+  storeConnectionData,
+} from './library/connections-adapter'
+import { ConnectionRegistry } from './services/connection-registry'
+import { WebSocketServerAdapter } from './library/web-socket-server-adapter'
+import { SubscriptionRegistry } from './services/subscription-registry'
+import {
+  deleteAllSubscriptions,
+  deleteSubscription,
+  fetchSubscriptions,
+  subscribeToReadModel,
+} from './library/subscription-adapter'
 
 export * from './paths'
 export * from './services'
 
 const eventRegistry = new EventRegistry()
 const readModelRegistry = new ReadModelRegistry()
+const connectionRegistry = new ConnectionRegistry()
+const subscriptionRegistry = new SubscriptionRegistry()
```

Comment:
If we apply the above suggestion, then we can do the following here:
```suggestion
const connectionRegistry = new SimpleRegistry(connectionsDatabase)
const subscriptionRegistry = new SimpleRegistry(subscriptionDatabase)
```